src/benchmarklib/benchmark_table_encoder.cpp

```diff
@@ -1,5 +1,6 @@
 #include "benchmark_table_encoder.hpp"
 
+#include <syncstream>
```

Comment:
Please sort alphabetically.

Reply:
But then the linter complains with `src/benchmarklib/benchmark_table_encoder.cpp:4:  Found C system header after C++ system header. Should be: benchmark_table_encoder.h, c system, c++ system, other.  [build/include_order] [4]`.

Reply:
Can you please push "this error" so that I can see the CI issue?